packages/components/pie-button/rollup.config.js

```diff
@@ -0,0 +1,22 @@
+import resolve from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+import { terser } from 'rollup-plugin-terser';
+
+// `npm run build` -> `production` is true
+// `npm run dev` -> `production` is false
+const production = !process.env.ROLLUP_WATCH;
+
+export default {
+	input: './compiled/index.js',
+	output: {
+		name: 'PieButton',
+		file: 'dist/bundle.js',
+		format: 'iife', // immediately-invoked function expression — suitable for <script> tags
+		sourcemap: false
```

Comment:
Again, not something to worry about too much yet, but should we just switch this on?